pkg/config/cos.go

```diff
@@ -537,6 +537,11 @@ func CreateRootPartitioningLayout(devPath string) (*yipSchema.YipConfig, error)
 		return nil, err
 	}
 
+	resolvedDevPath, err := filepath.EvalSymlinks(devPath)
```

Comment:
Does `EvalSymlinks` also work for normal device files?

Reply:
Yes. If it's not a symlink it would just return the original value.